server/vcs/provider/github/request/handler.go

```diff
@@ -297,10 +298,11 @@ func (h *Handler) handleCheckSuiteEvent(ctx context.Context, e *github.CheckSuit
 func (h *Handler) handlePullRequestReviewEvent(ctx context.Context, e *github.PullRequestReviewEvent, r *http.BufferedRequest) error {
 	pullRequestReviewEvent, err := h.pullRequestReviewEventConverter.Convert(e)
 	if err != nil {
+		h.logger.ErrorContext(ctx, "error parsing prr event", map[string]interface{}{"err": err.Error()})
```

Comment:
we should use the actual key definition when we do this, I've seen issues with typing that result in missing structured logs.